### DISCUSSION TITLE
How to connect "formatDetails" (for .gff) to plugin on JBrowse 2 desktop version.

### DISCUSSION THREAD

**Discussion by Chi-HsienChang** [GMOD/jbrowse-components#3617] [Q&A] [answered]
Hi JBrowse2 developers,

I want to ask: How to connect "formatDetails" (for .gff) to plugin on JBrowse 2 desktop version.

Here's I currently try do it, but the plugin does not work on JBrowse 2 **desktop version**.

1. Click to "Settings"
![image](https://user-images.githubusercontent.com/114167559/228866908-e75c345a-bc34-4d7a-85fd-3c7706b7dbcc.png)

2. In the FeatureTrack settings, add the jexl function to feature and subfeatures regions
![image](https://user-images.githubusercontent.com/114167559/228867653-1f2df452-1c55-458b-9015-2d3d7274541b.png)

3.  Go to the Plugin store and add the custom plugin by
Plugin name: Linkout
Plugin URL: https://unpkg.com/jbrowse-plugin-linkout/dist/jbrowse-plugin-linkout.umd.production.min.js
![image](https://user-images.githubusercontent.com/114167559/228868439-7830cd7d-5e95-448e-88a7-09af70db84cc.png)

4. Result:
![image](https://user-images.githubusercontent.com/114167559/228873914-f168f157-49fb-4fe1-9e19-91face7b6975.png)

It can not connect the plugin I installed it. 

Thank you!

**Comment by cmdcolin** [comment, marked as answer]
it's a little confusing UI-wise, but you have to side scroll over to check this box

![image](https://user-images.githubusercontent.com/6511937/228884926-cc71cd5e-7663-4584-aeee-ab5eb8675730.png)

this let's the slot know that you are going to paste code

then you just put

`{dbxref:linkout(feature)}`

in the text field

**Reply by Chi-HsienChang** [reply]
Hi @cmdcolin,

Wow, it works now!

Thank you so much!!!!!
